rust/connlib/shared/src/messages/client.rs

```diff
@@ -212,6 +212,14 @@ impl ResourceDescription {
             }
         }
     }
+
+    fn is_internet_resource(&self) -> bool {
+        if let ResourceDescription::Internet(_) = self {
+            return true;
+        }
+
+        false
```

Comment:
Surprised clippy didn't suggest something like
```suggestion
        matches!(self, ResourceDescription::Internet(_))
```